### PR TITLE
Docker opentelemetry

### PR DESCRIPTION
**Pull request by Davleen14**
Added otel collector config file and made changes to setup otel collector in docker compose. I tested it using `nodejs app`. You can refer the docs for setup of sample nodejs app. 
Update the link in app to be - 
```
const exporterOptions = {
    url: 'http://localhost:8081/otlp/v1/traces',
  }
```
Screenshot 2024-01-05 at 4 02 50 PM
Screenshot 2024-01-05 at 4 03 16 PM
Screenshot 2024-01-05 at 4 04 48 PM

### CODE REVIEW

File: .github/workflows/publish-prod-images.yml [hunk 1]
```diff
@@ -187,4 +187,14 @@ jobs:
             asset_path: docker-compose.yml
             asset_name: docker-compose.yml
             asset_content_type: application/octet-stream
+        -- name: Upload Release Asset - otel-collector-config.yaml
```
Comment (nkunal) [inline .github/workflows/publish-prod-images.yml:190]: there is a extra `-` here ?

File: docker-compose.yml [hunk 2]
```diff
@@ -13,3 +13,11 @@ services:
       - UI_PORT=${UI_PORT}
       - WORK_DIR=${WORK_DIR}
       - SIGLENS_VERSION=${SIGLENS_VERSION}
+
+  otel-collector:
```
Comment (nkunal) [inline docker-compose.yml:17]: we should name this as `siglens-otel-collector` and use that reference everywhere

File: otel-collector-config.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (nkunal) [inline otel-collector-config.yaml:1]: rename this file to be `siglens-otel-collector-config.yaml`

**Comment by nkunal** [comment]
@sunitakawane please review this PR

**Comment by sunitakawane** [comment]
these changes have been merged as part of other PR

**Pull request closed without merging**